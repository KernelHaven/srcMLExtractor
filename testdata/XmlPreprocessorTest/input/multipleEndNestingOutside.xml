<unit language="C">
	<block>
		<block>
			<cpp:ifdef>#<cpp:directive>ifdef</cpp:directive><name>A</name></cpp:ifdef>
			<empty_stmt>;</empty_stmt>
			<cpp:ifdef>#<cpp:directive>ifdef</cpp:directive><name>B</name></cpp:ifdef>
			<empty_stmt>;</empty_stmt>
		</block>
	</block>
	<cpp:endif>#<cpp:directive>endif</cpp:directive></cpp:endif>
	<cpp:endif>#<cpp:directive>endif</cpp:directive></cpp:endif>
</unit>
