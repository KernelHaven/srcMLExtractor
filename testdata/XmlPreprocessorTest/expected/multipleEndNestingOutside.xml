<unit language="C">
	<block>
		<block>
			<cpp:ifdef lineStart="3" lineEnd="10">#<cpp:directive>ifdef</cpp:directive><name>A</name>
				<empty_stmt>;</empty_stmt>
				<cpp:ifdef lineStart="5" lineEnd="9">#<cpp:directive>ifdef</cpp:directive><name>B</name>
					<empty_stmt>;</empty_stmt></cpp:ifdef></cpp:ifdef>
		</block>
	</block>


</unit>
